Readme.rst

```diff
@@ -63,8 +63,8 @@ In the root directory of the development distribution, you can do the following
 To run tests, run ``./bin/phpunit -c ./Build/BuildEssentials/PhpUnit/UnitTests.xml`` for unit or ``./bin/phpunit -c ./Build/BuildEssentials/PhpUnit/FunctionalTests.xml`` for functional/integration tests.
 
 To switch the branch you intend to work on:
-``git checkout 3.3 && composer update``
+``git checkout 7.1 && composer update``
 
-.. note:: We use an upmerging strategy, so create all bugfixes to lowest maintained branch that contains the issue (typically the second last LTS release, which is 3.3 currently), or master for new features.
+.. note:: We use an upmerging strategy, so create all bugfixes to lowest maintained branch that contains the issue (typically the second last LTS release, which is 4.3 currently), or master for new features.
```

Comment:
```suggestion
.. note:: We use an upmerging strategy, so create all bugfixes to lowest maintained branch that contains the issue (typically the second last LTS release, which is 5.3 currently), or master for new features.
```

Only true for another 4 days 😂 By the time, the master gets released, its already 5.3.

Reply:
Yeah but if I understand @albe right the Release will be more mid April ;)
But then the text does not fit ... because the 7.3 release take some time and if 4.3 is obsolete we have 2 releases without 2 maintained LTS versions ;)

Reply:
We have 5.3 as next LTS?

Referencing to: 

![image](https://user-images.githubusercontent.com/642226/112616367-ef9d6000-8e23-11eb-83fd-c9ac6935183a.png)

Reply:
Yes but the text say `typically the second last LTS release` and as 5.3 is the current LTS until 7.3 the second last is still 4.3 ;)

Reply:
ah, IMHO the text is wrong - at least it does not fit to the current practice.

Reply:
The "typically" was exactly for the case where we only have one previous LTS in active bugfix support, which happens every other year. Maybe need to revisit the release schedule/lifecycle at some point?